core/src/software/aws/toolkits/core/telemetry/TelemetryBatcher.kt

```diff
@@ -14,33 +15,31 @@ import java.util.concurrent.TimeUnit
 import java.util.concurrent.atomic.AtomicBoolean
 
 interface TelemetryBatcher {
-    fun enqueue(event: MetricEvent)
+    fun enqueue(event: MetricEvent) {
```

Comment:
I think the kotlin idiomatic way to do this would just be as an extension function.

Reply:
We actually only used the collection one in tests, so removed it and this is now the only method